### PR TITLE
fix(storefront): BCTHEME-940 optimize usage of jsTree library

### PR DESCRIPTION
**Pull request by BC-Chkalov-Valerii**
**What?**
This PR changes jsTree dependency from github to npm.
Also, optimized style bundling by moving jsTree-related styles into separate chunk.

**Tickets / Documentation**
[https://jira.bigcommerce.com/browse/BCTHEME-940](https://jira.bigcommerce.com/browse/BCTHEME-940)

**Screenshots (if appropriate)**
image

How it was before:
image

How it is now:
image

### CODE REVIEW

**Review by yurytut1993 — APPROVED** [review]
Lgtm. But can you please also add description to your pr like [here](https://github.com/bigcommerce/cornerstone/pull/2144) and update changelog

**Review by BC-krasnoshapka — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "bigcommerce-cornerstone",
   "description": "The BigCommerce reference theme for the Stencil platform",
-  "version": "6.1.3",
+  "version": "6.1.4",
```
Comment (BC-krasnoshapka) [inline package.json:4]: @BC-Chkalov-Valerii you don't need to bump version here, it is done by Stencil during release process.

**Review by BC-krasnoshapka — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Draft
 
+## 6.1.4 (17-11-2021)
```
Comment (BC-krasnoshapka) [inline CHANGELOG.md:9]: @BC-Chkalov-Valerii please keep Draft here as we don't know what version will be released.

File: package.json (hunk 1 above)
Comment (yurytut1993) [reply package.json:4]: @BC-krasnoshapka maybe it is acceptable if package related to ticket. In this case it looks more complete and we have more chances to find an issue during manual testing (imho)

**Review by BC-krasnoshapka — APPROVED** [review]

**Review by yurytut1993 — APPROVED** [review]

**Review by bc-azvierieva — APPROVED** [review]

**Pull request merged**